unminify.js

```diff
@@ -8,19 +8,20 @@
  * 2. Remove check for development mode - we always want unminified files.
  * 3. Remove BannerPlugin support - we don't use it.
  * 4. Remove the 'min' suffix from the chunk loaded in the new `mainEntry` option.
- * 5. Hook into compilation later so we're running after Source Map generation.
+ * 5. Hook into compilation later so we're running after Source Map generation. (https://webpack.js.org/api/compilation-hooks/: PROCESS_ASSETS_STAGE_OPTIMIZE_INLINE) 
```

Comment:
@rjchow Sorry, could you share some details on what and why you change this file? I think prettier makes it harder to review this.

Reply:
😆 It is hard to review regardless of prettier or not!

Basically I had to apply a 3-way "merge" between
1. Original author's webpack v4 version: https://github.com/leftstick/unminified-webpack-plugin/commit/734953e370fd55a3d68bfe3a1425adcaf8103d2b (deleted version of index.js)
2. Our webpack v4 version (previous commit of this file)
3. Original author's webpack v5 version: https://github.com/leftstick/unminified-webpack-plugin/blob/master/support5.js

I'm no expert at webpack so I largely relied on the heuristic of "is it working correctly yet?" to guide me through this.

When you look at the original author's webpack v5 version I think that accounts for most of the changes, then I applied the changes documented in this header comment block.

Reply:
> When you look at the original author's webpack v5 version I think that accounts for most of the changes, then I applied the changes documented in this header comment block.

Thanks @rjchow. This makes sense to me!